{{ cookiecutter.repo_name }}/.gitlab-ci.yml

```diff
@@ -236,6 +235,7 @@ tests:
   rules:
     - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH
 
+{% if cookiecutter.docs != "No docs" %}
```

Comment:
I would rather move it to `rules` part of the `pages` stage

Reply:
Good idea!

Reply:
Done.